src/PVRFilmonData.cpp

```diff
@@ -333,6 +359,27 @@ PVR_ERROR PVRFilmonData::GetRecordings(ADDON_HANDLE handle) {
 	return PVR_ERROR_NO_ERROR;
 }
 
+PVR_ERROR PVRFilmonData::GetRecordingStreamProperties(const PVR_RECORDING* recording, PVR_NAMED_VALUE* properties, unsigned int* iPropertiesCount) {
+  P8PLATFORM::CLockObject lock(m_mutex);
+  std:string strRecordingFile = "";
+  m_recordings = filmonAPIgetRecordings();
+  for (std::vector<PVRFilmonRecording>::iterator it = m_recordings.begin(); it != m_recordings.end(); it++) {
```

Comment:
You easily could rewrite this as `for (const auto& FilMonRecording : m_recordings)` => C++11 
Also applies to other parts of your PR.